apisix/schema_def.lua

```diff
@@ -389,6 +389,24 @@ local upstream_schema = {
             description = "discovery type",
             type = "string",
         },
+        discovery_host = {
```

Comment:
Better to use `nacos_address` instead.
All these 3 fields should go under a separate field called `discovery_args`, as it is nacos specific.
And we need to check them only when discovery_type `nacos` is used.